### PR TITLE
fix/typo

### PR DESCRIPTION
**Pull request by Maeevick**
I'm currently translating this part in french, I found this typo, how do you want to process ? 
- We keep the PR open until I finish the french translation (long run) in case I may find other ones ?
- We handle it as soon as possible ?

As you want :wink:

### CODE REVIEW

**Comment by Kleidukos** [comment]
Since we don't have anything in production with this typo I suggest you finish looking for typos and we will merge the PR when you don't find any

**Comment by Maeevick** [comment]
> Since we don't have anything in production with this typo I suggest you finish looking for typos and we will merge the PR when you don't find any

Fine, thanks for your insight ❤️

**Comment by Maeevick** [comment]
FYI : I reach the end of file @Kleidukos, I will double check with an ultimate run from start to end, then I will remove the "draft" state.

It may be cool, if you have time @JonathanLorimer, you confirm or invalidate the supposition I did about Maps' subtitle  : 
- Is "Updating Values to a Map" supart of "Maps" ?
- Is "When to use Maps" supart of "Maps" ?
- Is "HashMaps" supart of "Maps" ?

Thanks in advance guys :heart:

**Comment by Maeevick** [comment]
I read it and didn't find anything else, I guess it's ready :smile:

**Review by tchoutri — APPROVED** [review]

**Review by Kleidukos — APPROVED** [review]

File: en/lessons/basics/containers.md [hunk 1]
```diff
@@ -662,7 +662,7 @@ on the key.
 
 Sometimes we want a list like container where we can have performant indexed
 access. In fact arrays are often the primitive sequential data structure in
-lots of popular programming langauges. Haskell has this kind of data structures
+lots of popular programming languages. Haskell has this kind of data structures
```
Comment (JonathanLorimer) [inline en/lessons/basics/containers.md]:
```suggestion
lots of popular programming languages. Haskell has this kind of data structure
```

**Review by JonathanLorimer — COMMENTED** [review]
Just an additional typo!

File: en/lessons/basics/containers.md (hunk 1 above)
Comment (Maeevick) [reply en/lessons/basics/containers.md]: :+1:

**Review by JonathanLorimer — APPROVED** [review]

**Pull request merged**